projects/jobe/Dockerfile

```diff
@@ -12,59 +12,58 @@ LABEL \
     org.opencontainers.image.documentation="https://github.com/trampgeek/jobeinabox" \
     org.opencontainers.image.source="https://github.com/trampgeek/jobeinabox"
 
-ARG TZ=Etc/GMT-0
+ARG TZ=Pacific/Auckland
 # Set up the (apache) environment variables
-ENV APACHE_RUN_USER www-data
-ENV APACHE_RUN_GROUP www-data
-ENV APACHE_LOG_DIR /var/log/apache2
-ENV APACHE_LOCK_DIR /var/lock/apache2
-ENV APACHE_PID_FILE /var/run/apache2.pid
-ENV LANG C.UTF-8
+ENV APACHE_RUN_USER=www-data
+ENV APACHE_RUN_GROUP=www-data
+ENV APACHE_LOG_DIR=/var/log/apache2
+ENV APACHE_LOCK_DIR=/var/lock/apache2
+ENV APACHE_PID_FILE=/var/run/apache2.pid
+ENV LANG=C.UTF-8
 
 # Copy apache virtual host file for later use
 COPY 000-jobe.conf /
 # Copy test script
 COPY container-test.sh /
-COPY include/* /usr/include/
-RUN mkdir -p /usr/local/lib/java
-COPY files/* /usr/local/lib/java/
-RUN chmod a+w /usr/local/lib/java
-
 
+# Mount secrets
 # Set timezone
 # Install extra packages
 # Redirect apache logs to stdout
 # Configure apache
 # Configure php
-# Get and install jobe - from RunestoneInteractive/jobe - contains our changes.
+# Get jobe
+# Handle jobe API keys
+# Install jobe
 # Clean up
-RUN ln -snf /usr/share/zoneinfo/"$TZ" /etc/localtime && \
+RUN --mount=type=secret,id=api_keys \
+    export API_KEYS=`cat /run/secrets/api_keys | tr '\n' ' '` && \
+    ln -snf /usr/share/zoneinfo/"$TZ" /etc/localtime && \
     echo "$TZ" > /etc/timezone && \
     apt-get update && \
     apt-get --no-install-recommends install -yq \
-    acl \
-    apache2 \
-    build-essential \
-    catch2 \
-    fp-compiler \
-    git \
-    libapache2-mod-php \
-    nodejs \
-    octave \
-    openjdk-18-jdk \
```

Comment:
I wonder which version of java we end up with if we use default-jdk???  This could be bad for CSAwesome.